avm/res/cognitive-services/account/main.bicep

```
@@ -394,6 +398,14 @@ resource cognitiveService_lock 'Microsoft.Authorization/locks@2020-05-01' = if (
   scope: cognitiveService
 }
 
+resource commitmentPlansRes 'Microsoft.CognitiveServices/accounts/commitmentPlans@2025-04-01-preview' = [
```

As per resource symbolic names convention
```suggestion
resource cognitiveService_commitmentPlans 'Microsoft.CognitiveServices/accounts/commitmentPlans@2025-04-01-preview' = [
```